tests/unit/wsi.cpp

```
@@ -2035,33 +2035,41 @@ TEST_F(NegativeWsi, SwapchainMaintenance1ExtensionAcquire) {
     swapchain_create_info.pNext = &present_scaling_info;
 
     m_errorMonitor->SetDesiredFailureMsg(kErrorBit, "VUID-VkSwapchainPresentScalingCreateInfoEXT-presentGravityX-07765");
-    present_scaling_info.presentGravityY = 1;
+    m_errorMonitor->SetAllowedFailureMsg("VUID-VkSwapchainPresentScalingCreateInfoEXT-presentGravityX-07772");
```

It would be good to have a comment about why these errors are allowed.

---

Added comment. Considered also to have a conditional validation and do not validate zero value on platforms that support gravity. No need to suppress the errors then. But this will disable part of the validation, at least on nvidia, and will require some profile trickery to have the same coverage.